### PR TITLE
global: move to context

### PR DESCRIPTION
**Pull request by miekg**
Move from golang.org/x/net/context to std lib's context.

Change done with:

for i in $(grep -l '/context' **/*.go); do sed -e 's|golang.org/x/net/context|context|' -i $i; echo $i; done
for i in **/*.go; do goimports -w $i; done

~~Removed 'go get' from Makefile as well.~~

### CODE REVIEW

File: pb/dns.pb.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ import fmt "fmt"
 import math "math"
 
 import (
-	context "golang.org/x/net/context"
+	context "context"
```
Comment (johnbelamaric) [inline pb/dns.pb.go:21]: This is a generated file, we shouldn't be updating it. Does the latest version of `protoc-gen-go` use `context`?

Comment (johnbelamaric) [reply pb/dns.pb.go:21]: Hmm: https://github.com/golang/protobuf/blob/master/protoc-gen-go/grpc/grpc.go#L56

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] global: move ..." ]
>johnbelamaric commented on this pull request.
>
>
>
>> @@ -18,7 +18,8 @@ import fmt "fmt"
> import math "math"
>
> import (
>-	context "golang.org/x/net/context"
>+	context "context"
>
>Hmm: https://github.com/golang/protobuf/blob/master/protoc-gen-go/grpc/grpc.go#L56

What I thought, hence the manual edit. Note we haven't really document (by means 
of a Makefile) on how to generate this file. It also doesn't change much...

**Comment by miekg** [comment]
Let's figure this out one we need to regen that file

**Pull request merged**